### ISSUE TITLE
"Awakenings help" command

### ISSUE THREAD

**Issue by RheingoldRiver** [TsubakiBotPad/pad-cogs#324]
~~blocked by https://github.com/TsubakiBotPad/pad-data-pipeline/issues/62 (jk that's already done)~~

Would show a help text for each of the awakenings that a card has, probably divided into 2 sections, one for normal awakenings and one for SAs

**Comment by RheingoldRiver** [comment]
An `^idmode newuser 1` setting to make this part of the regular id screen rotation would be pretty fantastic as well

**Comment by turtleworks** [comment]
Will show every awakening but use some kind of truncated text for subsequent copies of the same awakening
